src/http_util.rs

```diff
@@ -27,7 +28,7 @@ pub fn get_client() -> Client<Connector, hyper::Body> {
 
 // The CodeFetch message is used to load HTTP javascript resources and expects a
 // synchronous response, this utility method supports that.
-pub fn fetch_sync_string(module_name: &str) -> DenoResult<String> {
+pub fn fetch_sync_string(module_name: &str) -> DenoResult<(String, String)> {
```

Comment:
How about returning `DenoResult<(String, msg::MediaType)>` ?

Reply:
I took a look at this and I decided not to change it... within `http_util` it doesn't really have a concept of the enum yet, as that is decided in `deno_dir.rs`.  So a decision here, might not be _right_ when combined with other logic contained in `deno_dir.rs`, so I would like to leave it as `(String, String)`.